index.html

```diff
@@ -0,0 +1,580 @@
+            Cosmic
+            Extragalactic Polarization, reported their results in a scientific briefing at the Center for
+            Astrophysics
+            here on Monday and in a set of papers submitted to The Astrophysical Journal.
+            <br> <br>
+    </article>
+
+    <article class="inflation">
+        <h3>The Theory of Inflation</h3>
+        Astronomers have found evidence to support the theory of inflation, which explains how the universe expanded so uniformly and so quickly in the instant after the Big Bang 13.8 billion years ago.
+        <div class="container-b">
+        <div class="inflation-illustration-a">
+<ul>
+    <li><img class="solar" src="images/inflation-1.jpg" alt="big-bang"></li>
+    <li><b>THE UNIVERSE</b> is just under 14 billion years old. From our position in the Milky Way galaxy, we can observe a sphere that is now about 92 billion light-years across. But there's a mystery. Wherever we look, the universe has an even temperature.</li>
+<li><img class="solar" src="images/inflation-5.jpg" alt="big-bang"></li>
+<li><b>STILL NOT ENOUGH TIME</b> has not been enough time to carry coffee cups from place to place before they get cold. But if all the coffee cups were somehow filled from a single coffee pot, all at the same time, that might explain their even temperature.</li>
+</ul>
+        </div>
+        <div class="inflation-illustration-b">
+            <ul>
+                <li><img class="solar" src="images/inflation-2.jpg" alt="big-bang"></li>
+                <li><b>NOT ENOUGH TIME</b> The universe is not old enough for light to have traveled the vast distance from one side of the universe to the other, and there has not been enough time for scattered patches of hot and cold to mix into an even temperature.</li>
+            <li><img class="solar" src="images/inflation-6.jpg" alt="big-bang"></li>
+            <li><b>INFLATION</b> this problem. The theory proposes that, less than a trillionth of a second after the Big Bang, the universe expanded faster than the speed of light. Tiny ripples in the violently expanding energy field eventually grew into the large-scale structures of the universe.</li>
+            </ul>
+        </div>
+        <div class="inflation-illustration-c">
+            <ul>
+                <li><img class="solar-b" src="/images/inflation-3.jpg" alt="big-bang"></li>
+            <li><b>DISTANT COFFEE</b> At a smaller scale, imagine using a telescope to look a mile in one direction. You see a coffee cup, and from the amount of steam, you can estimate its temperature and how much it has cooled.</li>
+        <li><img class="solar-b" src="/images/inflation-7.jpg" alt="big-bang"></li>    
+    <li><b>FLUCTUATION</b> have now detected evidence of these ancient fluctuations in swirls of polarized light in the cosmic background radiation, which is energy left over from the early universe. These are gravitational waves predicted by Einstein.</li>    
+    </ul>
```

Comment:
Your image in this section seems not to be showing, I'd suggest you try removing the `/` before the image path in your `src`, it'll solve that